wikimedia/wikipedia-ios
# [YiR] Update slides copy

**Phabricator:** https://phabricator.wikimedia.org/T377899

### Notes
* Updates copy for YiR slides

### Test Steps
1. Log in to MediaWiki on your browser, set your preferred language to Spanish or Italian on https://www.mediawiki.org/wiki/Special:GlobalPreferences
2. Set your device to Italy or Spain
3. Turn on the dev settings toggle
4. Browse 7+ articles 
5. Background and foreground the app 
6. Verify the intro slide copy
7. Verify the collective editing slides copy and link
